CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This changelog keeps track of all changes to the Packs SDK. We follow convention
 
 ## [Unreleased]
 
+### Added
+
+- Internal change to execute response
```

Comment:
let's not put things here yet

Reply:
We at least have to have non-empty content under Unreleased otherwise the release tooling barfs. Keeps coming up, we should follow our normal flow that every change that will eventually need releasing has some content in here, it's fine if it's just boilerplate though.

Reply:
Yes my understanding is we should add stuff here generally if we plan on releasing? This is pretty vague